api/base_controller.go

```diff
@@ -394,3 +395,7 @@ func pageFromObjectList(ctx context.Context, objectList types.ObjectList, count,
 	}
 	return page
 }
+
+func trimObjectTypePrefix(objectType types.ObjectType) string {
+	return strings.TrimPrefix(string(objectType), types.Prefix)
```

Comment:
Not sure I understand your question

Reply:
When converting from EntityType(storage entity) to ObjectType(api entity) with the `ToObjectType()` method I add the prefix, yes, because all of our ObjectType values have this prefix.

I think it would be lead to a lot of inconsistencies and possibly bugs if we start relying on this `ToObjectType()` function later on and expect it to output "standard" `ObjectType` entities, but it does not.